src/OpenTracing/UnsupportedFormatException.cs

```diff
@@ -1,21 +1,39 @@
-﻿using System;
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="UnsupportedFormatException.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
 
 namespace OpenTracing
 {
+    using System;
+
     /// <summary>
-    /// This exception should be used when the provided format value is unknown or disallowed by the <see cref="ITracer"/>.
+    ///     This exception should be used when the provided format value is unknown or disallowed by the <see cref="ITracer" />
```

Comment:
See previous comment. That seems to be your specific coding style. There should be no indentation and no space before "/>".

Reply:
resolved